frontend/packages/dev-console/src/utils/__tests__/pipeline-utils.spec.ts

```diff
@@ -10,9 +10,8 @@ import { mockPipelinesJSON } from './pipeline-test-data';
 describe('pipeline-utils ', () => {
   it('For first pipeline there should be 2 stages of length [0:[1],1:[2]]', () => {
```

Comment:
Noticed one other thing - the test name should change as well.

Reply:
updated